kubernetes/pybot/overlays/prod/deployment.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pybot
+spec:
+  replicas: 2
+  template:
+    spec:
+      containers:
+        - name: app
+          image: operationcode/pybot:master
+          env:
+            - name: BACKEND_URL
+              value: https://api.operationcode.org
+            - name: BACKEND_USERNAME
+              value: pybot@operationcode.org     
+            - name: COMMUNITY_CHANNEL
+              value: G3MD48QTD
+            - name: MENTOR_CHANNEL
+              value: G04CRMCT4
+            - name: MODERATOR_CHANNEL
+              value: GDUQK263D
+            - name: PYBACK_HOST
+              value: pyback-lb-197482116.us-east-2.elb.amazonaws.com
```

Comment:
If I understand what this does correctly, I _think_ this would be the new URL of pybot now - which we create via the ingress rules - `pybot.k8s.operationcode.org`